### PR TITLE
roachtest: use in-memory storage for ORM/driver tests

### LINKED ISSUE #111687: roachtest: activerecord failed
roachtest.activerecord [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=artifacts#/activerecord) on master @ [a227d7867b16e38ecd1b0b61bb2688ac566eb6ee](https://github.com/cockroachdb/cockroach/commits/a227d7867b16e38ecd1b0b61bb2688ac566eb6ee):

```
(activerecord.go:57).func1: pq: changing cluster setting "kv.raft_log.synchronization.unsafe.disabled" may cause cluster instability or data corruption.
To confirm the change, run the following command before trying again:
   SET unsafe_setting_interlock_key = 'EdARig==';
test artifacts and logs in: /artifacts/activerecord/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12040022/activerecord/1696373495182/1696373670649)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*activerecord.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-32014

### LINKED ISSUE #111688: roachtest: asyncpg failed
roachtest.asyncpg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=artifacts#/asyncpg) on master @ [a227d7867b16e38ecd1b0b61bb2688ac566eb6ee](https://github.com/cockroachdb/cockroach/commits/a227d7867b16e38ecd1b0b61bb2688ac566eb6ee):

```
(asyncpg.go:55).func1: pq: changing cluster setting "kv.raft_log.synchronization.unsafe.disabled" may cause cluster instability or data corruption.
To confirm the change, run the following command before trying again:
   SET unsafe_setting_interlock_key = 'PBwprQ==';
test artifacts and logs in: /artifacts/asyncpg/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=16</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12040022/asyncpg/1696373557289/1696373749506)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*asyncpg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-32015

### LINKED ISSUE #111699: roachtest: gopg failed
roachtest.gopg [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=artifacts#/gopg) on master @ [a227d7867b16e38ecd1b0b61bb2688ac566eb6ee](https://github.com/cockroachdb/cockroach/commits/a227d7867b16e38ecd1b0b61bb2688ac566eb6ee):

```
(gopg.go:61).func1: pq: changing cluster setting "kv.raft_log.synchronization.unsafe.disabled" may cause cluster instability or data corruption.
To confirm the change, run the following command before trying again:
   SET unsafe_setting_interlock_key = 'sZXzKg==';
test artifacts and logs in: /artifacts/gopg/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12040022/gopg/1696375640720/1696375810269)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gopg.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-32025

### LINKED ISSUE #111700: roachtest: django failed
roachtest.django [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=artifacts#/django) on master @ [a227d7867b16e38ecd1b0b61bb2688ac566eb6ee](https://github.com/cockroachdb/cockroach/commits/a227d7867b16e38ecd1b0b61bb2688ac566eb6ee):

```
(django.go:55).func1: pq: changing cluster setting "kv.raft_log.synchronization.unsafe.disabled" may cause cluster instability or data corruption.
To confirm the change, run the following command before trying again:
   SET unsafe_setting_interlock_key = 'SyFYeg==';
test artifacts and logs in: /artifacts/django/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=16</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12040022/django/1696375992922/1696376160800)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*django.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-32026

### LINKED ISSUE #111702: roachtest: gorm failed
roachtest.gorm [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=artifacts#/gorm) on master @ [a227d7867b16e38ecd1b0b61bb2688ac566eb6ee](https://github.com/cockroachdb/cockroach/commits/a227d7867b16e38ecd1b0b61bb2688ac566eb6ee):

```
(gorm.go:43).func1: pq: changing cluster setting "kv.raft_log.synchronization.unsafe.disabled" may cause cluster instability or data corruption.
To confirm the change, run the following command before trying again:
   SET unsafe_setting_interlock_key = '+o8c4w==';
test artifacts and logs in: /artifacts/gorm/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12040022/gorm/1696377593605/1696377766385)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*gorm.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-32027

### LINKED ISSUE #111703: roachtest: hibernate-spatial failed
roachtest.hibernate-spatial [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=artifacts#/hibernate-spatial) on master @ [a227d7867b16e38ecd1b0b61bb2688ac566eb6ee](https://github.com/cockroachdb/cockroach/commits/a227d7867b16e38ecd1b0b61bb2688ac566eb6ee):

```
(hibernate.go:105).func1: pq: changing cluster setting "kv.raft_log.synchronization.unsafe.disabled" may cause cluster instability or data corruption.
To confirm the change, run the following command before trying again:
   SET unsafe_setting_interlock_key = 'BgUHrg==';
test artifacts and logs in: /artifacts/hibernate-spatial/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12040022/hibernate-spatial/1696377969027/1696378139728)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*hibernate-spatial.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-32028

### LINKED ISSUE #111709: roachtest: jasync failed
roachtest.jasync [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=artifacts#/jasync) on master @ [a227d7867b16e38ecd1b0b61bb2688ac566eb6ee](https://github.com/cockroachdb/cockroach/commits/a227d7867b16e38ecd1b0b61bb2688ac566eb6ee):

```
(jasyncsql.go:42).func1: pq: changing cluster setting "kv.raft_log.synchronization.unsafe.disabled" may cause cluster instability or data corruption.
To confirm the change, run the following command before trying again:
   SET unsafe_setting_interlock_key = '2Lv1lA==';
test artifacts and logs in: /artifacts/jasync/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12040022/jasync/1696378228693/1696378401859)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*jasync.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-32033

### LINKED ISSUE #111710: roachtest: knex failed
roachtest.knex [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=artifacts#/knex) on master @ [a227d7867b16e38ecd1b0b61bb2688ac566eb6ee](https://github.com/cockroachdb/cockroach/commits/a227d7867b16e38ecd1b0b61bb2688ac566eb6ee):

```
(assertions.go:333).Fail: 
	Error Trace:	github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/knex.go:48
	            				github.com/cockroachdb/cockroach/pkg/cmd/roachtest/tests/knex.go:139
	            				main/pkg/cmd/roachtest/test_runner.go:1141
	            				src/runtime/asm_amd64.s:1598
	Error:      	Received unexpected error:
	            	pq: changing cluster setting "kv.raft_log.synchronization.unsafe.disabled" may cause cluster instability or data corruption.
	            	To confirm the change, run the following command before trying again:
	            	
	            	   SET unsafe_setting_interlock_key = 'xmNd8A==';
	            	
	Test:       	knex
(require.go:1360).NoError: FailNow called
test artifacts and logs in: /artifacts/knex/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12040022/knex/1696378695274/1696378878210)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*knex.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-32034

### LINKED ISSUE #111711: roachtest: lib/pq failed
roachtest.lib/pq [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=artifacts#/lib/pq) on master @ [a227d7867b16e38ecd1b0b61bb2688ac566eb6ee](https://github.com/cockroachdb/cockroach/commits/a227d7867b16e38ecd1b0b61bb2688ac566eb6ee):

```
(libpq.go:45).func1: pq: changing cluster setting "kv.raft_log.synchronization.unsafe.disabled" may cause cluster instability or data corruption.
To confirm the change, run the following command before trying again:
   SET unsafe_setting_interlock_key = 'avqBFQ==';
test artifacts and logs in: /artifacts/lib/pq/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12040022/lib-pq/1696378946966/1696379116123)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*lib/pq.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-32035

### LINKED ISSUE #111712: roachtest: liquibase failed
roachtest.liquibase [failed](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=log) with [artifacts](https://teamcity.cockroachdb.com/buildConfiguration/Cockroach_Nightlies_RoachtestNightlyGceBazel/12040022?buildTab=artifacts#/liquibase) on master @ [a227d7867b16e38ecd1b0b61bb2688ac566eb6ee](https://github.com/cockroachdb/cockroach/commits/a227d7867b16e38ecd1b0b61bb2688ac566eb6ee):

```
(liquibase.go:47).func1: pq: changing cluster setting "kv.raft_log.synchronization.unsafe.disabled" may cause cluster instability or data corruption.
To confirm the change, run the following command before trying again:
   SET unsafe_setting_interlock_key = 'rItA0g==';
test artifacts and logs in: /artifacts/liquibase/run_1
```
<p>Parameters: <code>ROACHTEST_arch=amd64</code>
, <code>ROACHTEST_cloud=gce</code>
, <code>ROACHTEST_cpu=4</code>
, <code>ROACHTEST_encrypted=false</code>
, <code>ROACHTEST_fs=ext4</code>
, <code>ROACHTEST_localSSD=true</code>
, <code>ROACHTEST_ssd=0</code>
</p>
<details><summary>Help</summary>
<p>

See: [roachtest README](https://github.com/cockroachdb/cockroach/blob/master/pkg/cmd/roachtest/README.md)

See: [How To Investigate \(internal\)](https://cockroachlabs.atlassian.net/l/c/SSSBr8c7)

See: [Grafana](https://go.crdb.dev/roachtest-grafana/teamcity-12040022/liquibase/1696378949942/1696379120432)

</p>
</details>
/cc @cockroachdb/sql-foundations
<sub>

[This test on roachdash](https://roachdash.crdb.dev/?filter=status:open%20t:.*liquibase.*&sort=title+created&display=lastcommented+project) | [Improve this report!](https://github.com/cockroachdb/cockroach/tree/master/pkg/cmd/internal/issues)
</sub>

Jira issue: CRDB-32036

### PR DESCRIPTION
**Pull request by rafiss**
This also removes the usage of an unsafe cluster setting, which is no longer relevant for in-memory clusters.

Additionally, fix a few parts of the roachprod code that would assume that an on-disk store is being used.

fixes https://github.com/cockroachdb/cockroach/issues/111722
fixes https://github.com/cockroachdb/cockroach/issues/111721
fixes https://github.com/cockroachdb/cockroach/issues/111720
fixes https://github.com/cockroachdb/cockroach/issues/111719
fixes https://github.com/cockroachdb/cockroach/issues/111718
fixes https://github.com/cockroachdb/cockroach/issues/111717
fixes https://github.com/cockroachdb/cockroach/issues/111716
fixes https://github.com/cockroachdb/cockroach/issues/111715
fixes https://github.com/cockroachdb/cockroach/issues/111714
fixes https://github.com/cockroachdb/cockroach/issues/111712
fixes https://github.com/cockroachdb/cockroach/issues/111711
fixes https://github.com/cockroachdb/cockroach/issues/111710
fixes https://github.com/cockroachdb/cockroach/issues/111709
fixes https://github.com/cockroachdb/cockroach/issues/111703
fixes https://github.com/cockroachdb/cockroach/issues/111702
fixes https://github.com/cockroachdb/cockroach/issues/111700
fixes https://github.com/cockroachdb/cockroach/issues/111699
fixes https://github.com/cockroachdb/cockroach/issues/111688
fixes https://github.com/cockroachdb/cockroach/issues/111687

Release note: None

### CODE REVIEW

File: pkg/cmd/roachtest/option/options.go [hunk 1]
```diff
@@ -30,6 +30,15 @@ func DefaultStartOpts() StartOpts {
 	return startOpts
 }
 
+// DefaultStartOptsInMemory returns a StartOpts populated with default values,
+// and with in-memory storage
+func DefaultStartOptsInMemory() StartOpts {
+	startOpts := StartOpts{RoachprodOpts: roachprod.DefaultStartOpts()}
+	startOpts.RoachprodOpts.ScheduleBackups = true
+	startOpts.RoachprodOpts.ExtraArgs = append(startOpts.RoachprodOpts.ExtraArgs, "--store=type=mem,size=0.3")
```
Comment (knz) [inline pkg/cmd/roachtest/option/options.go:38]: Maybe add a comment to explain "0.3 means 30% of available RAM".

**Review by knz — APPROVED** [review]
nice thanks

File: pkg/cmd/roachtest/option/options.go [hunk 2]
```diff
@@ -30,6 +30,15 @@ func DefaultStartOpts() StartOpts {
 	return startOpts
 }
 
+// DefaultStartOptsInMemory returns a StartOpts populated with default values,
+// and with in-memory storage
+func DefaultStartOptsInMemory() StartOpts {
+	startOpts := StartOpts{RoachprodOpts: roachprod.DefaultStartOpts()}
+	startOpts.RoachprodOpts.ScheduleBackups = true
```
Comment (renatolabs) [inline pkg/cmd/roachtest/option/options.go:37]: Nit: can these two lines be `startOpts = DefaultStartOpts()`?

File: pkg/cmd/roachtest/option/options.go (hunk 1 above)
Comment (rafiss) [reply pkg/cmd/roachtest/option/options.go:38]: done

**Comment by rafiss** [comment]
tftr!

bors r=knz

**Comment by rafiss** [comment]
bors r=knz

**Review by renatolabs — APPROVED** [review]
Thanks for fixing the roachtest support for in-memory storage!

**Review by RaduBerinde — APPROVED** [review]

**Pull request merged**